CHANGELOG.md

```diff
@@ -176,7 +176,9 @@ GM seed.
 
 ### Enhancements
 
-* None.
+* Add `SyncPermissionOffer` and `SyncPermissionOfferResponse` classes to allow
```

Comment:
Thanks. I moved to the enhancements section in next version. I also add sync breaking change entry for changing primary key of `PermissionChange`